cosmos/operators/local.py

```diff
@@ -248,6 +252,29 @@ def store_compiled_sql(self, tmp_project_dir: str, context: Context, session: Se
         else:
             logger.info("Warning: ti is of type TaskInstancePydantic. Cannot update template_fields.")
 
+    @provide_session
+    def store_freshness_json(self, tmp_project_dir: str, context: Context, session: Session = NEW_SESSION) -> None:
```

Comment:
shall we add some test for this method?

Reply:
Do you have any suggestions for this?
I'm not 100% sure how to access the rendered template of a node after a run (if not using the UI)

Reply:
I added some unit tests